aspnet/mvc/views/forms/sample/final/ViewModels/RegisterViewModelFinal.cs

```diff
@@ -0,0 +1,20 @@
+﻿//#define final
+#if final
```

Comment:
Even if condition is useful, no need for two files (and their strange names). Just place the `using` and the attributes in conditional blocks.